### PR TITLE
Fixing typos, wording corrections

### PR DESCRIPTION
**Pull request by dkrees**

### CODE REVIEW

File: pages/categories/upstream.md [hunk 1]
```diff
@@ -25,17 +25,17 @@ Emissions associated with developing and delivering commercial off-the-shelf (CO
...
+Emissions from manufacturing, transport, and installation of all the hardware equipment used within the organisation's technology infrastructure. This spans employee devices, networking gear, and on-premise data centres:
 
 
 {% include linkedHeading.html heading="Employee Hardware" level=3 %}
 Laptops, desktops, mobiles, printers and peripherals used by employees
 
 
 {% include linkedHeading.html heading="Networking Hardware" level=3 %}
-Network routers, switches, wifi access points installed
+Network routers, switches, wi-fi access points installed
```
Comment (nickbarber) [inline pages/categories/upstream.md:36]: i think Wi-Fi should be capitialised technically. Here and on operational

**Review by nickbarber — CHANGES_REQUESTED** [review]
Only other issues i spotted are on the About page there's an open source that needs hyphenating to open-source and it's needs changing to its at "Scott Logic has aligned it’s approach"

**Comment by kdavis04** [comment]
Minor comment - there's a few places particularly in upstream where the sentences under the subheadings don't end in full stops, whereas they do in the other categories

**Comment by nickbarber** [comment]
> Only other issues i spotted are on the About page there's an open source that needs hyphenating to open-source and it's needs changing to its at "Scott Logic has aligned it’s approach"

@dkrees did you see this comment

**Review by nickbarber — APPROVED** [review]

**Pull request merged**